teams/teams-ps/teams/Set-CsTeamsMessagingPolicy.md

```diff
@@ -25,8 +25,9 @@ Set-CsTeamsMessagingPolicy [-Tenant <Guid>] [-Description <String>] [-AllowUrlPr
  [-AllowStickers <Boolean>] [-AllowUserTranslation <Boolean>] [-AllowImmersiveReader <Boolean>]
  [-AllowRemoveUser <Boolean>] [-AllowPriorityMessages <Boolean>] [-AllowSmartReply <Boolean>] [-Allow [-ReadReceiptsEnabledType <String>]
  [-AudioMessageEnabledType <AudioMessageEnabledTypeEnum>] [-ChannelsInChatListEnabledType <ChannelsInChatListEnabledTypeEnum>]
- [[-Identity] <XdsIdentity>] [-Force] [-WhatIf] [-Confirm] [<CommonParameters>] [-ChatPermissionRole <String>] [-AllowSmartCompose] <Boolean>]
- [-CreateCustomEmojis <Boolean>] [-DeleteCustomEmojis <Boolean>]
+ [-CreateCustomEmojis <Boolean>] [-DeleteCustomEmojis <Boolean>] [-ChatPermissionRole <String>] [-AllowSmartCompose] <Boolean>]
+ [-DesignerForBackgroundsAndImages <DesignerForBackgroundsAndImagesTypeEnum>]
+ [[-Identity] <XdsIdentity>] [-Force] [-WhatIf] [-Confirm] [<CommonParameters>]
```

Comment:
I reordered here so identify, force and the rest at the bottom.
Also is the extra [] around -Identity a typo?

Reply:
This is [[-Identity] <XdsIdentity>] is used in other places so I would leave it as is.